CITATION.cff

```diff
@@ -34,6 +37,9 @@ authors:
     family-names: Young
     affiliation: University College London
     orcid: 'https://orcid.org/0000-0002-8432-7769'
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.7950604
```

Comment:
Uhm, the example at https://citation-file-format.github.io/  shows something like
```suggestion
doi: 10.5281/zenodo.7950604
```

Reply:
This is what the tool gave me. 🤷 
I do prefer yours though because: shorter.